nodes/mesecons.lua

```diff
@@ -69,10 +82,14 @@ for d = 0, 1 do
 			formspec = wrench.META_TYPE_STRING,
 			real_portstates = wrench.META_TYPE_INT,
 		},
+	description = function(pos, meta, node, player)
+		return meta:get_string("infotext")
+	end,
 	})
 end
 end
 end
 end
 
+luacontroller_defs.drop = nil
```

Comment:
Wouldn't it be better to change it to an unburnt Luacontroller? 🤔

Reply:
If wrench is meant to allow moving nodes unmodified then I think no but it is more about making decision what wrench is.

Reply:
Moving as is burnt is burnt allows better compatibility with some corner cases and afaik there's no losses: memory is already cleared and luac can be jus reprogrammed.
Really corner cases like customization mod that requires some items for repairing burnt lua controller, bit far fetched but as decision how tool should consistently behave it might be better.